apl/aggregation-function/dcount.mdx

```diff
@@ -7,6 +7,10 @@ The `dcount` aggregation function in Axiom Processing Language (APL) counts the
 
 Use `dcount` for analyzing datasets where it’s important to identify the number of distinct occurrences, such as unique IP addresses in security logs, unique user IDs in application logs, or unique trace IDs in OpenTelemetry traces.
 
+<Note>
+The `dcount` aggregation in APL is a statistical aggregation that gives estimated results. The estimation comes with the benefit of speed at the expense of accuracy. This means that `dcount` is fast and light on resources even on a large or high-cardinality dataset, but it doesn’t provide the most accurate results.
```

Comment:
"doesn’t provide the most accurate results" is not very confidence-inspiring :sweat_smile: , maybe better like this?
```suggestion
The `dcount` aggregation in APL is a statistical aggregation that gives estimated results. The estimation comes with the benefit of speed at the expense of accuracy. This means that `dcount` is fast and light on resources even on a large or high-cardinality dataset, but it doesn’t provide precise results.
```